client/lib/api/who_date_format.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:intl/intl.dart';
+
+extension WHOFormat on DateTime {
+  String get whoFormat => DateFormat.yMMMd().format(
+        // Timstamps are for midnight. It looks as though the WHO dashboard uses the date of when the data was reported to WHO rather than the original day.
```

Comment:
The comment is missing the crucial information:
```
// Timestamps are for midnight so the day it applies to is ambiguous.
// Add 1 day to match the WHO Dashboard: http://covid19.who.int
```
The information about it being day reported rather than the day it happened should be a separate comment.

Reply:
@brunobowden changes made